components/profile-controller/main.go

```diff
@@ -43,35 +43,30 @@ func init() {
 	_ = clientgoscheme.AddToScheme(scheme)
 
 	_ = profilev1.AddToScheme(scheme)
-	_ = istiorbac.AddToScheme(scheme)
+	_ = istioSecurityClient.AddToScheme(scheme)
 	// +kubebuilder:scaffold:scheme
 }
 
 func main() {
-	var metricsAddr, leaderElectionNamespace string
+	var metricsAddr string
 	var enableLeaderElection bool
 	var userIdHeader string
 	var userIdPrefix string
 	var workloadIdentity string
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
-	flag.StringVar(&leaderElectionNamespace, "leader-election-namespace", "",
-		"Determines the namespace in which the leader election configmap will be created.")
 	flag.StringVar(&userIdHeader, USERIDHEADER, "x-goog-authenticated-user-email", "Key of request header containing user id")
 	flag.StringVar(&userIdPrefix, USERIDPREFIX, "accounts.google.com:", "Request header user id common prefix")
 	flag.StringVar(&workloadIdentity, WORKLOADIDENTITY, "", "Default identity (GCP service account) for workload_identity plugin")
-
 	flag.Parse()
 
 	ctrl.SetLogger(zap.Logger(true))
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                  scheme,
-		MetricsBindAddress:      metricsAddr,
-		LeaderElection:          enableLeaderElection,
-		LeaderElectionNamespace: leaderElectionNamespace,
-		LeaderElectionID:        "kubeflow-profile-controller",
```

Comment:
Why are the options `LeaderElectionNamespace` and `LeaderElectionID` removed?

Reply:
That is my bad - I did a copy-paste of a file - let me fix that.